python/lsst/ts/MTAlignment/alignmentModel.py

```diff
@@ -32,20 +32,20 @@ def __init__(self, host, port, log=logging.getLogger()):
         self.simulation_mode = 0
         self.com_lock = asyncio.Lock()
 
-    async def connect(self):
+    async def connect(self, host, port):
```

Comment:
Why pass the `host` and `port` here instead of using `self.host` and `self.port`? this is a bit confusing.